.devcontainer/Dockerfile

```
@@ -0,0 +1,32 @@
+ARG VARIANT=latest
+FROM elixir:${VARIANT}
```

```suggestion
FROM elixir:latest
```
I installed vscode and tried to boot this up and it installed elixir 1.7. Hard coding this as latest fixed it for me.

---

Ah thanks for the heads up, I made a change to the config file to used `latest` also added a setup in the guide calling this out if people would like to set a specific version of Elixir